### PR TITLE
[Doc] update RESUME/STOP ROUTINE LOAD

### PR DESCRIPTION
**Pull request by hellolilyliuyi**
Why I'm doing:

What I'm doing:

Fixes #issue

## What type of PR is this:

- [ ] BugFix
- [ ] Feature
- [ ] Enhancement
- [ ] Refactor
- [ ] UT
- [x] Doc
- [ ] Tool

Does this PR entail a change in behavior?

- [ ] Yes, this PR will result in a change in behavior.
- [x] No, this PR will not result in a change in behavior.

If yes, please specify the type of change:

- [ ] Interface/UI changes: syntax, type conversion, expression evaluation, display information
- [ ] Parameter changes: default values, similar parameters but with different default values
- [ ] Policy changes: use new policy to replace old one, functionality automatically enabled
- [ ] Feature removed
- [ ] Miscellaneous: upgrade & downgrade compatibility, etc.

## Checklist:

- [ ] I have added test cases for my bug fix or my new feature
- [ ] This pr needs user documentation (for new or modified features or behaviors)
  - [ ] I have added documentation for my new feature or new function
- [ ] This is a backport pr

## Bugfix cherry-pick branch check:
- [x] I have checked the version labels which the pr will be auto-backported to the target branch
  - [x] 3.2
  - [x] 3.1
  - [x] 3.0
  - [x] 2.5

### CODE REVIEW

**Review by yingtingdong — DISMISSED** [review]

File: docs/en/sql-reference/sql-statements/data-manipulation/RESUME_ROUTINE_LOAD.md [hunk 1]
```diff
@@ -4,10 +4,27 @@ displayed_sidebar: "English"
...
 # RESUME ROUTINE LOAD
 
+## Description
+
+Resume a Routine load job. The job will temporarily enter **NEED_SCHEDULE** state because the job is being re-scheduled. And after some time, the job will be resumed to **RUNNING** state, continuing consuming messages from the data source and loading data. You can check the job's information with the [SHOW ROUTINE LOAD](https://docs.starrocks.io/docs/3.2/sql-reference/sql-statements/data-manipulation/SHOW_ROUTINE_LOAD/) statement.
+
+## Syntax
+
+```SQL
+RESUME ROUTINE LOAD FOR [db_name.]<job_name>
```
Comment (rickif) [inline docs/en/sql-reference/sql-statements/data-manipulation/RESUME_ROUTINE_LOAD.md:14]:
```suggestion
RESUME ROUTINE LOAD FOR [db_name].<job_name>
```

File: docs/en/sql-reference/sql-statements/data-manipulation/STOP_ROUTINE_LOAD.md [hunk 2]
```diff
@@ -4,10 +4,34 @@ displayed_sidebar: "English"
...
+
+- A stopped Routine Load job can not be resumed.
+- If you only need to pause the Routine Load job, you can execute [PAUSE ROUTINE LOAD](./PAUSE _ROUTINE_LOAD.md).
+
+:::
+
+## Syntax
+
+```SQL
+STOP ROUTINE LOAD FOR [db_name.]<job_name>
```
Comment (rickif) [inline docs/en/sql-reference/sql-statements/data-manipulation/STOP_ROUTINE_LOAD.md:21]:
```suggestion
STOP ROUTINE LOAD FOR [db_name].<job_name>
```

File: docs/zh/sql-reference/sql-statements/data-manipulation/RESUME_ROUTINE_LOAD.md [hunk 3]
```diff
@@ -6,12 +6,25 @@ displayed_sidebar: "Chinese"
 
 ## 功能
 
-恢复已暂停 routine load 导入任务，通过 [PASUE](../data-manipulation/PAUSE_ROUTINE_LOAD.md) 命令可以暂停导入的任务，并进行 routine load 任务属性的修改，详细操作请参考 [ALTER ROUTINE LOAD](./ALTER_ROUTINE_LOAD.md)。
+恢复 Routine Load 导入作业。导入作业会先短暂地进入 **NEED_SCHEDULE** 状态，表示正在重新调度导入作业，一段时间后会恢复至 **RUNNING** 状态，继续消费数据源的消息并且导入数据。您可以执行 [SHOW ROUTINE LOAD](./SHOW_ROUTINE_LOAD.md) 语句查看已恢复的导入作业。
+
+## 语法
+
+```SQL
+RESUME ROUTINE LOAD FOR [db_name.]<job_name>
```
Comment (rickif) [inline docs/zh/sql-reference/sql-statements/data-manipulation/RESUME_ROUTINE_LOAD.md:14]:
```suggestion
RESUME ROUTINE LOAD FOR [db_name].<job_name>
```

File: docs/en/sql-reference/sql-statements/data-manipulation/PAUSE_ROUTINE_LOAD.md [hunk 4]
```diff
@@ -15,14 +15,14 @@ This statement pauses a Routine Load job but does not terminate this job. You ca
 ## Syntax
 
 ```SQL
-PAUSE ROUTINE LOAD FOR <db_name>.<job_name>;
+PAUSE ROUTINE LOAD FOR [db_name.]<job_name>;
```
Comment (evelynzhaojie) [inline docs/en/sql-reference/sql-statements/data-manipulation/PAUSE_ROUTINE_LOAD.md:18]: the privilege requirement in line 13 is not correct, it's the privilege name in v2.5 and earier

File: docs/en/sql-reference/sql-statements/data-manipulation/PAUSE_ROUTINE_LOAD.md [hunk 5]
```diff
@@ -15,14 +15,14 @@ This statement pauses a Routine Load job but does not terminate this job. You ca
...
 ```
 
 ## Parameters
 
 | Parameter | Required | Description                                                  |
 | --------- | -------- | ------------------------------------------------------------ |
-| db_name   |          | The name of the database for which you want to pause a Routine Load job. |
+| db_name   |          | The name of the database where the Routine Load job belongs. |
```
Comment (amber-create) [inline docs/en/sql-reference/sql-statements/data-manipulation/PAUSE_ROUTINE_LOAD.md:25]: image
-> to which

File: docs/en/sql-reference/sql-statements/data-manipulation/RESUME_ROUTINE_LOAD.md [hunk 6]
```diff
@@ -4,10 +4,27 @@ displayed_sidebar: "English"
 
 # RESUME ROUTINE LOAD
 
+## Description
+
+Resume a Routine load job. The job will temporarily enter **NEED_SCHEDULE** state because the job is being re-scheduled. And after some time, the job will be resumed to **RUNNING** state, continuing consuming messages from the data source and loading data. You can check the job's information with the [SHOW ROUTINE LOAD](https://docs.starrocks.io/docs/3.2/sql-reference/sql-statements/data-manipulation/SHOW_ROUTINE_LOAD/) statement.
```
Comment (amber-create) [inline docs/en/sql-reference/sql-statements/data-manipulation/RESUME_ROUTINE_LOAD.md:9]: image
-> Resumes

File: docs/en/sql-reference/sql-statements/data-manipulation/RESUME_ROUTINE_LOAD.md [hunk 7]
```diff
@@ -4,10 +4,27 @@ displayed_sidebar: "English"
...
+
+```SQL
+RESUME ROUTINE LOAD FOR [db_name.]<job_name>
+```
+
+## Parameters
+
+| **Parameter** | **Required** | **Description**                                              |
+| ------------- | ------------ | ------------------------------------------------------------ |
+| db_name       |              | The name of the database where the Routine Load job belongs. |
```
Comment (amber-create) [inline docs/en/sql-reference/sql-statements/data-manipulation/RESUME_ROUTINE_LOAD.md:21]: image
-> to which

File: docs/en/sql-reference/sql-statements/data-manipulation/STOP_ROUTINE_LOAD.md [hunk 8]
```diff
@@ -4,10 +4,34 @@ displayed_sidebar: "English"
 
 # STOP ROUTINE LOAD
 
+## Description
+
+Stop a Routine Load job.
```
Comment (amber-create) [inline docs/en/sql-reference/sql-statements/data-manipulation/STOP_ROUTINE_LOAD.md:9]: image
-> Stops

File: docs/en/sql-reference/sql-statements/data-manipulation/STOP_ROUTINE_LOAD.md [hunk 9]
```diff
@@ -4,10 +4,34 @@ displayed_sidebar: "English"
...
+
+::: warning
+
+- A stopped Routine Load job can not be resumed.
```
Comment (amber-create) [inline docs/en/sql-reference/sql-statements/data-manipulation/STOP_ROUTINE_LOAD.md:13]: image
-> cannot

File: docs/en/sql-reference/sql-statements/data-manipulation/STOP_ROUTINE_LOAD.md [hunk 10]
```diff
@@ -4,10 +4,34 @@ displayed_sidebar: "English"
...
+
+```SQL
+STOP ROUTINE LOAD FOR [db_name.]<job_name>
+```
+
+## Parameters
+
+| **Parameter** | **Required** | **Description**                                              |
+| ------------- | ------------ | ------------------------------------------------------------ |
+| db_name       |              | The name of the database where the Routine Load job belongs. |
```
Comment (amber-create) [inline docs/en/sql-reference/sql-statements/data-manipulation/STOP_ROUTINE_LOAD.md:28]: image
-> to which

File: docs/zh/sql-reference/sql-statements/data-manipulation/STOP_ROUTINE_LOAD.md [hunk 11]
```diff
@@ -6,12 +6,33 @@ displayed_sidebar: "Chinese"
 
 ## 功能
 
-停止 routine load 任务，停止了的导入任务会在间隔一段时间后自动删除。
+停止导入 Routine Load 作业。
```
Comment (amber-create) [inline docs/zh/sql-reference/sql-statements/data-manipulation/STOP_ROUTINE_LOAD.md:9]: image”
“导入”的位置错了吧？应该放在“作业”前面

**Review by amber-create — DISMISSED** [review]

File: docs/zh/sql-reference/sql-statements/data-manipulation/PAUSE_ROUTINE_LOAD.md [hunk 12]
```diff
@@ -4,28 +4,32 @@ displayed_sidebar: "Chinese"
 
 # PAUSE ROUTINE LOAD
 
+import RoutinelLoadPrivNote from '../assets/commonMarkdown/RoutinelLoadPrivNote.md'
```
Comment (evelynzhaojie) [inline docs/zh/sql-reference/sql-statements/data-manipulation/PAUSE_ROUTINE_LOAD.md:7]: RoutinelLoadPrivNote.md  这个名称不对，Routine 后面多了一个 l?

File: docs/en/sql-reference/sql-statements/data-manipulation/RESUME_ROUTINE_LOAD.md [hunk 13]
```diff
@@ -4,10 +4,31 @@ displayed_sidebar: "English"
 
 # RESUME ROUTINE LOAD
 
+import RoutineLoadPrivNote from '../assets/commonMarkdown/RoutineLoadPrivNote.md'
+
+## Description
+
+Resumes a Routine load job. The job will temporarily enter **NEED_SCHEDULE** state because the job is being re-scheduled. And after some time, the job will be resumed to **RUNNING** state, continuing consuming messages from the data source and loading data. You can check the job's information with the [SHOW ROUTINE LOAD](https://docs.starrocks.io/docs/3.2/sql-reference/sql-statements/data-manipulation/SHOW_ROUTINE_LOAD/) statement.
```
Comment (evelynzhaojie) [inline docs/en/sql-reference/sql-statements/data-manipulation/RESUME_ROUTINE_LOAD.md:11]: with the -- using the

**Review by evelynzhaojie — DISMISSED** [review]

**Review by EsoragotoSpirit — DISMISSED** [review]

**Review by amber-create — DISMISSED** [review]

**Review by evelynzhaojie — APPROVED** [review]

**Review by EsoragotoSpirit — APPROVED** [review]

**Comment by hellolilyliuyi** [comment]
@mergify backport  branch-3.2 branch-3.1 branch-3.0 branch-2.5

**Pull request merged**